### ISSUE TITLE
Error creating H2 for Geonames

### ISSUE THREAD

**Issue by fhoubie** [malesch/geonames-h2#1]
Hi, I get this error when I try to create a H2 instance. Where can I see the reason ?
Thanks

geonames-h2.core=> (create-geonames-db :cities1000)
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
19-06-12 14:21:15 LEU-XFHOUBIE01 INFO [geonames-h2.core:116] - Download `cities1000.txt` from `http://download.geonames.org/export/dump/cities1000.zip`
19-06-12 14:21:17 LEU-XFHOUBIE01 INFO [geonames-h2.core:118] - Import `/Users/fhoubie/Dev/github/geonames-h2/download/cities1000.txt` into table `cities1000`
19-06-12 14:21:17 LEU-XFHOUBIE01 INFO [geonames-h2.core:41] - Create table `cities1000`
19-06-12 14:21:17 LEU-XFHOUBIE01 ERROR [geonames-h2.core:124] - Exception processing table `cities1000`: nil
                    clojure.core/eval   core.clj: 3105
                                  ...
            geonames-h2.core/eval6725  REPL Input
                                  ...
  geonames-h2.core/create-geonames-db   core.clj:  151
  geonames-h2.core/create-geonames-db   core.clj:  160
          geonames-h2.core/import-all   core.clj:  115
       geonames-h2.core/import-all/fn   core.clj:  119
geonames-h2.core/perform-table-import   core.clj:  106
        geonames-h2.core/create-table   core.clj:   46
                 clojure.core/ex-info   core.clj: 4617
clojure.lang.ExceptionInfo: SQL table cannot be created
    reason: "Wrong number of args (20) passed to: jdbc/create-table-ddl"
     table: :cities1000

**Comment by malesch** [comment]
Salut Frédéric

I couldn't reproduce your issue but can it be that you have updated the org.clojure/java.jdbc dependency? I noticed that the table creation syntax has changed and was deprecated in release 0.5.6 and finally no more available from release 0.6.0-alpha1 on. 

See for more details the issue [JDBC-120](https://clojure.atlassian.net/browse/JDBC-120) and the [Change Log](https://github.com/clojure/java.jdbc/tree/java.jdbc-0.5.6).

I updated the JDBC dependency and fixed the table creation.
